torquedata/core/jinja_factory.py

```diff
@@ -0,0 +1,12 @@
+from jinja2 import Environment
```

Comment:
I probably would have just put all of this in utils.py, unless you can see this file growing substantially bigger.

Though maybe we should really split utils into many smaller files so the imports are cleaner in the target directories.

Reply:
Also a good call!  I'll do that as well (splitting utils might make sense but in the mean time I will stick with the pattern of the project!)